cypress/e2e/shared/editor.test.ts

```diff
@@ -91,12 +94,33 @@ describe('Editor+LSP communication', () => {
   })
 
   describe('in Script Editor', () => {
+    const setScriptToFlux = () => {
+      return cy.isIoxOrg().then(isIox => {
+        if (isIox) {
+          cy.getByTestID('query-builder--new-script')
+            .should('be.visible')
+            .click()
+          cy.getByTestID('script-dropdown__flux').should('be.visible').click()
+          cy.getByTestID('overlay--container').within(() => {
+            cy.getByTestID('flux-query-builder--no-save').click({force: true})
+          })
```

Comment:
Is the element obscured by something else on the DOM here? Can we check for `visible` or do a `scrollIntoView()` instead of force clicking?

Reply:
Fixed in the next PRs (where this PR is broken into two).